lib/config/config.js

```diff
@@ -103,6 +104,30 @@ class Config {
       gatewayWatcher.isReady = true;
     });
   }
+
+  // this function suppose to preserve comments.
+  // due to fragile nature of YAWN it may not work properly.
+  updateGently (modifier) {
+    return new Promise((resolve, reject) => {
+      fs.readFile(this.gatewayConfigPath, 'utf8', (err, data) => {
+        if (err) {
+          return reject(err);
+        }
+        let json = yamlOrJson.load(data);
```

Comment:
There should be `const` instead of `let`. (`let` is a signal that the variable may be reassigned)

Reply:
`eslint` is usually able to catch these things - is it configured in CircleCI?